### ISSUE TITLE
Title ignores vertical gravity

### ISSUE THREAD

**Issue by amethystant** [faruktoptas/FancyShowCaseView#205]
Used FancyShowCaseView 1.3.1

I'm focusing on a button in a fragment. The focus is right, but the title always stays on top of the screen no matter what gravity I set. It doesn't ignore horizontal gravity though, if I set it to CENTER, it centres the text but only horizontally. Vertical position never changes. It's not affected by what view I focus on and it's not affected by fitSystemWindows. I managed to overcome this by using a custom view instead, but it might be worth looking into. 

Example:
In Fragment onStart() method I use the code below to show:
```kotlin
requireView().findViewById<View>(R.id.report_button).let {
    FancyShowCaseView.Builder(requireActivity())
        .focusOn(it)
        .title(getString(R.string.report))
        .titleGravity(Gravity.BOTTOM)
        .build()
        .show()
}
```

**Comment by faruktoptas** [comment]
Thank you @amethystant. I've seen the bug. It will be fixed with the next release. You can also use `enableAutoTextPosition()` parameter to align the title vertically to the center of the bigger space.

**Comment by jama5262** [comment]
@faruktoptas Wondering if this bug is fixed

**Comment by faruktoptas** [comment]
@jama5262 It's not released yet. I'll release it asap.

**Comment by faruktoptas** [comment]
@amethystant @jama5262 this bug is fixed with `1.3.2` version.

**Comment by Dixzz** [comment]
Hello, I'm facing similar issue, the "text" is not horizontally centered nor Gravity.END is working
FancyShowCaseView version: 1.3.2


```
new FancyShowCaseView.Builder(app)
                            .focusOn((findViewById(R.id.nav_main)))
                            .focusShape(FocusShape.ROUNDED_RECTANGLE)
                            .roundRectRadius(24)
                            .fitSystemWindows(true)
                            .title("text")
                            .titleStyle(R.style.focusTitle, Gravity.CENTER_HORIZONTAL | Gravity.BOTTOM)
                            .build()
                            .show();

<style name="focusTitleStyle">
        <item name="android:textSize">@dimen/text_18sp</item>
        <item name="android:textColor">@color/colorPrimary</item>
        <item name="android:fontFamily">@font/google_sans_bold</item>
</style>
```
![Screenshot_1600623392](https://user-images.githubusercontent.com/33489151/93717844-04a7e680-fb96-11ea-87ad-8ba437274667.png)
